src/com/clouway/fraction/fractionValidator.js

```
@@ -0,0 +1,13 @@
+/**
+ * Created by Stanislava on 06.06.16.
+ */
+function isFraction(fieldId) {
+    var fraction = document.getElementById(fieldId).value;
+
+    var pattern = new RegExp("^\[0-9]{1,10}([,.][0-9]{1,5})$");
+    if (!pattern.test(fraction)) {
+        document.getElementById("error").className = "fractions";
+    } else {
+        document.getElementById("success").className = 'fractions';
```

This way you will not be hiding the error message when the input value is corrected and both messages will be visible. (btw, the demo is not updated)
Change the class of one element in both validity cases, not 2 different elements.